classes/cluster/mk22_lab_advanced/stacklight/server.yml

```diff
@@ -38,3 +38,25 @@ parameters:
           enabled: true
           type: eth
           proto: dhcp
+  influxdb:
```

Comment:
you could put it into a system class like it's done for galera databases. Eg, system.influxdb.database.ceilometer and eventually the declaration of the StackLight database should move there too.